src/scancode/cli.py

```diff
@@ -109,6 +113,13 @@
 
 notice = acknowledgment_text.strip().replace('  ', '')
 
+scan_proper_plugins = PluginManager('scan_proper')
+scan_output_plugins = PluginManager('scan_output')
+scan_proper_plugins.add_hookspecs(hookspec)
+scan_output_plugins.add_hookspecs(hookspec)
+scan_proper_plugins.load_setuptools_entrypoints('scancode_plugins')
+scan_output_plugins.load_setuptools_entrypoints('scancode_plugins')
```

Comment:
I think that a format plugin is not a general "scancode_plugins" but rather a "scancode_format_plugin" of sorts (may be rather singular than plural?)